   Classe do erro: excecao.InfraestruturaException   Metodo: inclui   Mensagem: java.lang.NullPointerException   Stack Trace: excecao.InfraestruturaException: java.lang.NullPointerException
	at dao.controle.InterceptadorDeDAO.intercept(InterceptadorDeDAO.java:46)
	at dao.impl.DocumentoDAOImpl$$EnhancerByCGLIB$$2dea3128.inclui(<generated>)
	at servico.impl.DocumentoAppServiceImpl.inclui(DocumentoAppServiceImpl.java:26)
	at servico.impl.DocumentoAppServiceImpl$$EnhancerByCGLIB$$bdde30ab.CGLIB$inclui$1(<generated>)
	at servico.impl.DocumentoAppServiceImpl$$EnhancerByCGLIB$$bdde30ab$$FastClassByCGLIB$$ab752dde.invoke(<generated>)
	at net.sf.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:215)
	at servico.controle.InterceptadorDeServico.intercept(InterceptadorDeServico.java:46)
	at servico.impl.DocumentoAppServiceImpl$$EnhancerByCGLIB$$bdde30ab.inclui(<generated>)
	at Principal.inclui_aroundBody2(Principal.java:42)
	at Principal$AjcClosure3.run(Principal.java:1)
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)
	at aspecto.AspectoParaLogDeErros.efetuaLog(AspectoParaLogDeErros.java:27)
	at Principal.main(Principal.java:42)
Caused by: java.lang.NullPointerException
	at dao.impl.DocumentoDAOImpl.inclui(DocumentoDAOImpl.java:22)
	at dao.impl.DocumentoDAOImpl$$EnhancerByCGLIB$$2dea3128.CGLIB$inclui$1(<generated>)
	at dao.impl.DocumentoDAOImpl$$EnhancerByCGLIB$$2dea3128$$FastClassByCGLIB$$46e80c28.invoke(<generated>)
	at net.sf.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:215)
	at dao.controle.InterceptadorDeDAO.intercept(InterceptadorDeDAO.java:42)
	... 12 more
   Classe do erro: excecao.InfraestruturaException   Metodo: inclui   Mensagem: java.lang.NullPointerException   Stack Trace: excecao.InfraestruturaException: java.lang.NullPointerException
	at dao.controle.InterceptadorDeDAO.intercept(InterceptadorDeDAO.java:46)
	at dao.impl.DocumentoDAOImpl$$EnhancerByCGLIB$$823aa50a.inclui(<generated>)
	at servico.impl.DocumentoAppServiceImpl.inclui(DocumentoAppServiceImpl.java:26)
	at servico.impl.DocumentoAppServiceImpl$$EnhancerByCGLIB$$122ea48d.CGLIB$inclui$1(<generated>)
	at servico.impl.DocumentoAppServiceImpl$$EnhancerByCGLIB$$122ea48d$$FastClassByCGLIB$$1f26a890.invoke(<generated>)
	at net.sf.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:215)
	at servico.controle.InterceptadorDeServico.intercept(InterceptadorDeServico.java:46)
	at servico.impl.DocumentoAppServiceImpl$$EnhancerByCGLIB$$122ea48d.inclui(<generated>)
	at Principal.inclui_aroundBody2(Principal.java:42)
	at Principal$AjcClosure3.run(Principal.java:1)
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)
	at aspecto.AspectoParaLogDeErros.efetuaLog(AspectoParaLogDeErros.java:27)
	at Principal.main(Principal.java:42)
Caused by: java.lang.NullPointerException
	at dao.impl.DocumentoDAOImpl.inclui(DocumentoDAOImpl.java:22)
	at dao.impl.DocumentoDAOImpl$$EnhancerByCGLIB$$823aa50a.CGLIB$inclui$1(<generated>)
	at dao.impl.DocumentoDAOImpl$$EnhancerByCGLIB$$823aa50a$$FastClassByCGLIB$$bfa2fa3.invoke(<generated>)
	at net.sf.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:215)
	at dao.controle.InterceptadorDeDAO.intercept(InterceptadorDeDAO.java:42)
	... 12 more
2018-11-03T18:14:22.602:  Classe do erro: excecao.InfraestruturaException   Metodo: inclui   Mensagem: java.lang.NullPointerException   Stack Trace: excecao.InfraestruturaException: java.lang.NullPointerException
	at dao.controle.InterceptadorDeDAO.intercept(InterceptadorDeDAO.java:46)
	at dao.impl.DocumentoDAOImpl$$EnhancerByCGLIB$$823aa50a.inclui(<generated>)
	at servico.impl.DocumentoAppServiceImpl.inclui(DocumentoAppServiceImpl.java:26)
	at servico.impl.DocumentoAppServiceImpl$$EnhancerByCGLIB$$122ea48d.CGLIB$inclui$0(<generated>)
	at servico.impl.DocumentoAppServiceImpl$$EnhancerByCGLIB$$122ea48d$$FastClassByCGLIB$$1f26a890.invoke(<generated>)
	at net.sf.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:215)
	at servico.controle.InterceptadorDeServico.intercept(InterceptadorDeServico.java:46)
	at servico.impl.DocumentoAppServiceImpl$$EnhancerByCGLIB$$122ea48d.inclui(<generated>)
	at Principal.inclui_aroundBody2(Principal.java:42)
	at Principal$AjcClosure3.run(Principal.java:1)
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)
	at aspecto.AspectoParaLogDeErros.efetuaLog(AspectoParaLogDeErros.java:28)
	at Principal.main(Principal.java:42)
Caused by: java.lang.NullPointerException
	at dao.impl.DocumentoDAOImpl.inclui(DocumentoDAOImpl.java:22)
	at dao.impl.DocumentoDAOImpl$$EnhancerByCGLIB$$823aa50a.CGLIB$inclui$0(<generated>)
	at dao.impl.DocumentoDAOImpl$$EnhancerByCGLIB$$823aa50a$$FastClassByCGLIB$$bfa2fa3.invoke(<generated>)
	at net.sf.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:215)
	at dao.controle.InterceptadorDeDAO.intercept(InterceptadorDeDAO.java:42)
	... 12 more
